web/client/src/main.css

```
@@ -29,26 +29,80 @@
+  --color-warning: #ffc66b;
+  --color-warning-content: #1b1103;
+
+  --color-neutral: #dce5f0;
+  --color-neutral-content: #101722;
+
+  --color-main-border: #34445a;
+  --color-chart-grid: #41526a;
+  --color-login-pattern: #172538;
 }
+
 @import './animations.css';
```

Stylelint (17.14.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)